app/src/main/java/com/github/se/travelpouch/ui/home/TravelList.kt

```diff
@@ -85,6 +87,11 @@ fun TravelListScreen(
               Icon(imageVector = Icons.Default.Add, contentDescription = "Add")
             }
       },
+      bottomBar = {
+        BottomNavigationMenu(
+            tabList = listOf(TopLevelDestinations.NOTIFICATION, TopLevelDestinations.TRAVELS),
+            navigationActions = navigationActions)
+      },
```

Comment:
Be careful, this may not work on the small phone in landscape mode. I would recommand you to reduce the size of the map first, if you want to use this method